packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Handle OnClick
+ *
+ * @param {*} e event object
+ */
```

Comment:
The documentation on this seems very light. The code itself seems to be looking for the event object's target href then doing something with it. I think there needs to be more instruction on what this common utility is doing and what you need to set it to. Include an `@example` too.

Reply:
Okay.